libbeat/template/field_test.go

```diff
@@ -1,325 +0,0 @@
-package template
```

Comment:
the tests are mostly moved over to `template/processor_test`, tests regarding unpacking are moved to `common/field_test.go`